### ISSUE TITLE
i have 50 mocha files in my test folder and i am running the below command to run all tests at once.How can i get one report for one mocha file seperately .mocha tests--timeout 6000000 --reporter mochawesome --reporter-options reportDir=%batchexecutionresultspath%,reportFilename=TestReport,overwrite=false,timestamp

### ISSUE THREAD

**Issue by TSSumanth** [adamgruber/mochawesome-report-generator#122]

**Comment by TSSumanth** [comment]
The %batchexecutionresultspath% path should contain all my test results separately with unique name.
Any suggestions.

**Comment by adamgruber** [comment]
That's not how mocha works. If you pass it a directory, it will run all the tests in that directory as a single test suite. If you need a separate test run and report for each file then you would have to use some kind of script that would call `mocha` on each file.

**Comment by TSSumanth** [comment]
@adamgruber do u mean creating a batch file that iterates over the tests folder and execute each file in it??

**Comment by adamgruber** [comment]
Yeah something like that. It's the only way I know of that would create a separate report for each test file. Wanting a separate report for each file kind of defeats the purpose of the reporter though and mocha for that matter.
